doc/release-notes-27460.md

```diff
@@ -0,0 +1,4 @@
+- A new `importmempool` RPC has been added. It loads a `mempool.dat` file (previously created via
```

Comment:
Is it only `mempool.dat` file saved from the `savemempool` RPC? I think a valid `mempool.dat` that was dumped after a node shutdown can also be imported?
```suggestion
A new `importmempool` RPC has been added. It loads a valid `mempool.dat` file and attempts to add its contents to the mempool. This can be useful to
  import mempool data from another node without having to modify the datadir contents and restart the node.
```

Reply:
You're right, I dropped the `savemempoool` as suggested. Thanks!